docs/database/database-metrics.md

```diff
@@ -239,6 +241,176 @@ and SHOULD be provided **at span creation time** (if provided at all):
 
 ## Experimental
 
+### Database Response
+
+The following metric instruments describe database query response.
+
+#### Metric: `db.client.response.row_count`
+
+This metric is [required][MetricRequired].
+
+This metric SHOULD be specified with
+[`ExplicitBucketBoundaries`](https://github.com/open-telemetry/opentelemetry-specification/tree/v1.35.0/specification/metrics/api.md#instrument-advisory-parameters)
+of `[1, 10, 50, 100, 250, 500, 1000, 2000, 5000, 10000]`.
```

Comment:
I'd recommend following https://github.com/open-telemetry/semantic-conventions/issues/316#issuecomment-1783528869

Reply:
I tried to explain here, why I came up with this bucket configuration. Here we divided buckets into 4 different categories and assigned boundaries to them. I am not sure if it is applied to all the metrics.

Can you please provide your suggestion on this bucket config based on the algorithm you provided.

Reply:
changed it to `b`